src/main/kotlin/app/revanced/patches/music/misc/microg/fingerprints/ServiceCheckFingerprint.kt

```diff
@@ -0,0 +1,22 @@
+package app.revanced.patches.music.misc.microg.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patcher.fingerprint.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patches.music.misc.microg.annotations.MusicMicroGPatchCompatibility
+import org.jf.dexlib2.AccessFlags
+
+@Name("google-play-service-checker-fingerprint")
+@MatchingMethod(
+    "Lnuv;", "d"
+)
+@FuzzyPatternScanMethod(2) // FIXME: Test this threshold and find the best value.
+@MusicMicroGPatchCompatibility
+@Version("0.0.1")
+object ServiceCheckFingerprint : MethodFingerprint(
+    "V", AccessFlags.PUBLIC or AccessFlags.STATIC, listOf("L", "I"), listOf(null,

```

Comment:
Why do you have a listOf(null) here?

Reply:
the string was enough and i decided to not use opcodes

Reply:
Then use null instead of listof(null)

Reply:
done